spring-cloud-dataflow-docs/src/main/asciidoc/applications.adoc

```diff
@@ -0,0 +1,8 @@
+[[applications]]
+= Applications
+
+[partintro]
+--
+A selection of pre-built link:http://cloud.spring.io/spring-cloud-stream-app-starters/[stream] and link:http://cloud.spring.io/spring-cloud-task-app-starters/[task/batch] starter apps for various data integration and processing scenarios facilitate learning and experimentation. For more details, review how to <<streams.adoc#spring-cloud-dataflow-register-stream-apps, register applications>>
```

Comment:
I was just about to say that `file:///home/mpollack/projects/spring-cloud-dataflow/spring-cloud-dataflow-docs/target/generated-docs/index.html#spring-cloud-dataflow-register-stream-apps` has this info.... so we should link to that section instead of pointing back (only) to the spring-cloud-stream-apps project page.